packages/zoe/src/isOfferSafe.js

```diff
@@ -23,46 +21,18 @@ import { assert, details } from '@agoric/assert';
  * order as the issuers for the payoutRules. This array of amount is the
  * reallocation to be given to a player.
  */
-function isOfferSafeForOffer(amountMaths, payoutRules, amounts) {
-  assert(
-    amountMaths.length === payoutRules.length &&
-      amountMaths.length === amounts.length,
-    details`amountMaths, payoutRules, and amounts must be arrays of the same length`,
-  );
-
-  const allowedRules = ['offerAtMost', 'wantAtLeast'];
-
-  for (const payoutRule of payoutRules) {
-    if (payoutRule === null || payoutRule === undefined) {
-      throw new Error(`payoutRule must be specified`);
-    }
-    assert(
-      allowedRules.includes(payoutRule.kind),
-      details`The kind ${payoutRule.kind} was not recognized`,
-    );
-  }
-
+function isOfferSafeForOffer(amountMaths, offerRules, newAmounts) {
```

Comment:
Can this file be renamed to `offerSafety.js`?  Or does javascript style call for a file to be named the same as its exports when there's only one?

Reply:
It could be renamed to offerSafety. I don't think the style is that strict about filenames, but I could be wrong. If we do rename it though, it's companion file is `areRightsConserved` so that should also be renamed to something like `rightsConservation`

Reply:
These files are renamed